src/Backend/Modules/Pages/Domain/Page/Page.php

```diff
@@ -252,6 +252,57 @@ public function __construct(
         $this->allowDelete = $allowDelete;
     }
 
+    public function update(
+        int $id,
+        int $userId,
+        ?int $parentId,
+        ?int $templateId,
+        Meta $meta,
+        string $language,
+        string $title,
+        string $navigationTitle,
+        DateTime $publishOn,
+        int $sequence,
+        bool $navigationTitleOverwrite = false,
+        bool $hidden = true,
+        string $status = self::ACTIVE,
+        string $type = 'root',
+        $data = null,
+        bool $allowMove = true,
+        bool $allowChildren = true,
+        bool $allowEdit = true,
+        bool $allowDelete = true
+    ): void {
+        $this->id = $id;
+        $this->userId = $userId;
+        $this->parentId = $parentId;
+        if ($this->parentId === null) {
+            $this->parentId = 0;
+        }
+        $this->templateId = $templateId;
+        if ($this->templateId === null) {
+            $this->templateId = 0;
```

Comment:
can't you just do this by casting the parentId to int?

Reply:
This is more explicit.